platform/darwin/src/MGLStyle.mm

```diff
@@ -293,7 +293,7 @@ - (void)insertObject:(MGLStyleLayer *)styleLayer inLayersAtIndex:(NSUInteger)ind
         }
```

Comment:
I agree with you: https://github.com/mapbox/mapbox-gl-native/pull/7484#discussion_r93326377. Unfortunately it's this comment that stuck around. 😁